src/lighteval/logging/evaluation_tracker.py

```diff
@@ -281,6 +303,31 @@ def push_to_hub(
 
         self.recreate_metadata_card(repo_id)
 
+    def push_results_to_hub(self, repo_id: str, path_in_repo: str, private: bool | None = None):
```

Comment:
Shouldn't `repo_id` be `results` by default ?

Reply:
Same for `path_in_repo` - do we want to follow as default what we had? (org/modelname/results_date.json)